code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm

```diff
@@ -191,6 +191,12 @@
 /mob/living/simple_animal/drone/cogscarab/ratvar_act()
 	fully_heal(TRUE)
 
+/mob/living/simple_animal/drone/cogscarab/can_speak_in_language(datum/language/dt)
```

Comment:
The only mobs that have this boilerplate is drones and cogscarabs. AIs are more complicated.

Also, I felt that putting it in the language would be... counter intuitive? Given it's the mob code that  determines whether a language can be spoken or not.